README.md

```diff
@@ -1,5 +1,3 @@
-
-
```

Comment:
Would be great to add "Read this in Hindi" as a link here, just like the "read this in Portuguese" link below this line.